rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I think so - https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/setup-maas.yml#L25 would be the line modified, which is filtered before getting down to the role.

In this case, it might be better to do `hosts:all_containers:!compute_hosts`...though are there specific compute host checks that get put down?